common/constants.js

```diff
@@ -198,11 +198,6 @@ CONSTANTS.SCRIPT_LIBRARIES = {
   },
 };
 
-CONSTANTS.REQUEST_BODY_TYPES = {
-  STATIC: "static",
-  CODE: "code",
-};
-
```

Comment:
why delete these?

Reply:
This is a duplicate, there's another definition of this constant just below this